src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/ResponsibilityCommentTests.kt

```diff
@@ -96,4 +105,49 @@ class ResponsibilityCommentTests : DatabaseTest() {
             assertThat(it).isEqualTo("OK")
         }
     }
+
+    @Test
+    fun `can get responsibilities data for touchstone`(){
+        JooqContext().use {
+            it.setupResponsibilities()
+        }
+
+        val responsibilitiesUrl = "/touchstones/touchstone-1/responsibilities/csv/"
+        val permissions = PermissionSet(
+                "*/can-login",
+                "*/touchstones.read",
+                "*/responsibilities.review"
+        )
+        val response = RequestHelper().get(responsibilitiesUrl, permissions, acceptsContentType = "text/csv")
+
+        assertThat(response.headers["Content-Type"]).isEqualTo("text/csv")
+        assertThat(response.headers["Content-Disposition"])
+                .isEqualTo("attachment; filename=\"responsibilities_touchstone-1.csv\"")
+
+        val schema = CSVSchema("ResponsibilitySetsWithComments")
+        schema.validate(response.text)
+
+        val csv = StringReader(response.text)
+                .use { CSVReader(it).readAll() }
+        val dataRow = csv.drop(1).first().toList()
```

Comment:
Could check the header row a well.

Reply:
That's done by the `schema.validate(response.text)` (L128)